run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
I only have command line access.

Is there a way to get these files on the command line?

Ben


On Sun, Feb 8, 2015 at 4:13 PM, dnath <notifications@github.com> wrote:

> In run.mac.sh
> <https://github.com/StochSS/stochss/pull/281#discussion_r24304503>:
>
> > @@ -1,5 +1,12 @@
> >  #!/usr/bin/env bash
> >
> > +osname=$(uname)
>
> @bbbales2 <https://github.com/bbbales2>, Try checking your aws s3 bucket
> at https://console.aws.amazon.com/s3/home?region=us-east-1 --- Can you
> access it? If not, there is some issue with granting S3 access to your creds
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StochSS/stochss/pull/281/files#r24304503>.
>